diff --git a/modules/user/user.module b/modules/user/user.module
index b239799..62ca1d1 100644
--- a/modules/user/user.module
+++ b/modules/user/user.module
@@ -2883,19 +2883,7 @@ function user_roles($membersonly = FALSE, $permission = NULL) {
 
   $roles = array();
   foreach ($result as $role) {
-    switch ($role->rid) {
-      // We only translate the built in role names
-      case DRUPAL_ANONYMOUS_RID:
-        if (!$membersonly) {
-          $roles[$role->rid] = t($role->name);
-        }
-        break;
-      case DRUPAL_AUTHENTICATED_RID:
-        $roles[$role->rid] = t($role->name);
-        break;
-      default:
-        $roles[$role->rid] = $role->name;
-    }
+    $roles[$role->rid] = $role->name;
   }
 
   return $roles;
